.dockerignore

```diff
@@ -1,4 +1,3 @@
 .git
 .next
-*Dockerfile*
-node_modules
\ No newline at end of file
+*Dockerfile*
```

Comment:
Why do we keep the node_modules as part of the context ? I thought that we should not keep it so that we do not upload node_modules to the docker image (which could be system specific), and so that we have a clean install inside the Docker build.

Reply:
Good point, I'll investigate, I think I had some kind of error and modified this, this might in fact might not be needed 🤔